source/special.tex

```diff
@@ -3203,17 +3203,23 @@
 \end{example}
 
 \pnum
-When the criteria for elision of a copy/move operation are met,
-but not for an \nonterminal{exception-declaration},
-and the object
-to be copied is designated by an lvalue,
-or when the \grammarterm{expression} in a \tcode{return} statement
+In the following copy-initialization contexts, a move constructor might be used instead of a copy constructor:
```

Comment:
I don't think this is entirely accurate: the constructor we end up using might not be a move constructor (in particular, it might be instantiated from a constructor template). Can we make this a little more vague (given that the precise rules are later)?